pkg/pgmodel/migrate.go

```diff
@@ -125,6 +125,18 @@ func CheckDependencies(db *pgxpool.Pool, versionInfo VersionInfo) (err error) {
 	return checkExtensionVersion(db)
 }
 
+func CheckDBVersion(ctx context.Context, conn *pgx.Conn, versionInfo VersionInfo) error {
```

Comment:
I think this needs to be re-written to use pgmodel.CheckDependencies instead?

Reply:
we deliberately don't want to check the extension as that can race, see the updated code